index.html

```diff
@@ -4853,6 +4853,67 @@ <h3>Issuer</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h4>Holder</h4>
+        <p>
+The value associated with the <code>holder</code> <a>property</a> is expected
+to identify the <a>holder</a> to the <a>verifier</a>.
+        </p>
+        <p>
+Relevant metadata about the <code>holder</code> <a>property</a> is expected
+to be available to the <a>verifier</a>. For example, a <a>holder</a> can
+publish information containing the verification material used to secure
+<a>verifiable presentations</a>. This metadata is relevant when
+checking proofs on <a>verifiable presentations</a>.
+        </p>
+        <p>
+See the <a data-cite="VC-IMP-GUIDE/#subject-holder-relationships"></a> and
+<a data-cite="VC-USE-CASES#user-tasks"></a> for additional examples related to <a>subject</a> and <a>holder</a>.
+        </p>
+
+        <p class="note">
+`Issuer`, `subject`, and `holder` are graph nodes which support multiple representations, 
+potentially making it complex to evaluate whether these roles are being filled
+by the same entity.
+        </p>
+        <ul>
+          <li>
+<a href="#issuer">Issuers</a> define expressions of `issuer` in <a>credentials</a>
+          </li>
+          <li>
+<a href="#presentations-0">Presentations</a> define expressions of `holder` in <a>presentations</a>
+          </li>
+          <li>
+<a href="#credential-subject">Credential subjects</a> define expressions of `credentialSubject` in <a>credentials</a>
+          </li>
+        </ul>
+        <p>
+A <a>verifier</a> might conjecture that a <a>verifiable presentation</a> <a>holder</a> 
+is the same entity as a <a>verifiable credential</a> <a>subject</a>, when the following conditions apply:
```

Comment:
This is not particularly rigorous language and probably doesn't belong in a specification. We shouldn't say anything about "what an [entity] might conjecture". Rather, we should describe the problem and possible approaches concretely.

```suggestion
A <a>verifier</a> might need to validate a given <a>verifiable presentation</a> against complex business rules. For example, when the verifier needs confidence that the <a>holder</a> is the same entity as a the  <a>subject</a> of <a>verifiable credential</a>. In that situation, the following factors should provide reasonable confidence that the holder presenting a given VC is, in fact, the subject of that VC:
```

Reply:
```suggestion
A <a>verifier</a> might need to validate a given <a>verifiable presentation</a> against complex business rules; for example, the verifier may need confidence that the <a>holder</a> is the same entity as a <a>subject</a> of a <a>verifiable credential</a>. In that situation, the following factors should provide reasonable confidence that the holder presenting a given VC is, in fact, a subject of that VC:
```